default/web_tt2/css.tt2

```diff
@@ -2743,3 +2744,4 @@ a, li {
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
 }
 
+/* end css.tt2 */
```

Comment:
CSS-style comments (at beginning and ending) seem not required, although they may not be harm.

Reply:
As there is a TT2 comment at the beginning of the file, I removed CSS comments in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.